### PR TITLE
event-model: more fixes

### PR DESCRIPTION
**Pull request by mrakitin**
This PR resolves the issue observed in #603: we don't limit the Python's version in the `build` section.

### CODE REVIEW

**Comment by CJ-Wright** [comment]
We usually do it in both but we also don't have a build section (we have a host section when compilers aren't involved)

File: recipes-tag/event-model/meta.yaml [hunk 1]
```diff
@@ -9,13 +9,13 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record=record.txt
   skip: True  # [py2k]
 
 requirements:
   build:
```
Comment (CJ-Wright) [inline recipes-tag/event-model/meta.yaml]:
```suggestion
  host:
```

**Comment by CJ-Wright** [comment]
Ah I see the issue this is not a noarch recipe (you can just move the selector to ``py>=36``)

**Comment by mrakitin** [comment]
Yeah, it's already arch...

> (you can just move the selector to `py>=36`)
what do you mean by that?

File: recipes-tag/event-model/meta.yaml (hunk 1 above)
Comment (mrakitin) [reply recipes-tag/event-model/meta.yaml]: I hope it won't break anything.

**Comment by danielballan** [comment]
Thanks @CJ-Wright!

File: recipes-tag/event-model/meta.yaml [hunk 2]
```diff
@@ -9,13 +9,13 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record=record.txt
   skip: True  # [py2k]
```
Comment (CJ-Wright) [inline recipes-tag/event-model/meta.yaml]:
```suggestion
  skip: True  # [py>=3.6]
```
From https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html#preprocessing-selectors

File: recipes-tag/event-model/meta.yaml [hunk 3]
```diff
@@ -9,18 +9,18 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record=record.txt
-  skip: True  # [py2k]
+  skip: True  # [py>=3.6]
```
Comment (mrakitin) [inline recipes-tag/event-model/meta.yaml]: Are we sure we skip the newer versions? I think it should be `py<3.6`...

File: recipes-tag/event-model/meta.yaml (hunk 2 above)
Comment (mrakitin) [reply recipes-tag/event-model/meta.yaml]: Thanks for the link! I think the condition should be opposite. See my comment below.

File: recipes-tag/event-model/meta.yaml (hunk 3 above)
Comment (mrakitin) [reply recipes-tag/event-model/meta.yaml]: OK, looks good now.

**Comment by mrakitin** [comment]
@CJ-Wright, thanks for a few commits, feel free to merge whenever you are ready.

**Comment by mrakitin** [comment]
OK, self-merge to have it built sooner.

**Comment by CJ-Wright** [comment]
sorry

**Pull request merged**